org-code-javabuilder/validation/src/main/java/org/code/validation/NeighborhoodLog.java

```diff
@@ -0,0 +1,96 @@
+package org.code.validation;
+
+public class NeighborhoodLog {
```

Comment:
Mind adding a comment at the top of the classes noting which are user-facing?

Reply:
will do

Reply:
I ended up splitting out the "support" files into a sub-package to make things clearer (I also added comments).